crates/pink/sidevm/env/src/lib.rs

```diff
@@ -0,0 +1,627 @@
+//! Tools for writing Sidevm programs.
+
+// #![warn(missing_docs)]
+
+use core::{
+    future::Future,
+    ops::{Deref, DerefMut},
+    pin::Pin,
+    task,
+};
+use std::cell::RefCell;
+
+use crate::ocall_funcs_guest as ocall;
+use num_enum::{IntoPrimitive, TryFromPrimitive};
+use scale::{Decode, Encode};
+use tinyvec::TinyVec;
+
+pub use args_stack::RetEncode;
+pub use ocall_def::*;
+pub use pink_sidevm_macro::main;
+pub use tasks::{spawn, TaskHandle};
+
+mod args_stack;
+mod ocall_def;
+mod tasks;
+
+cfg_if::cfg_if! {
+    if #[cfg(any(target_pointer_width = "32", feature = "host"))] {
+        pub type IntPtr = i32;
+    } else {
+        // For unit test
+        pub type IntPtr = i64;
+    }
+}
+
+pub type IntRet = i64;
```

Comment:
Since it's 64 bit integer, how about call it `Int64Ret` directly? It made me a bit confused when reading args_stack.rs

Reply:
Sounds good